docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -0,0 +1,93 @@
+[[tune-for-indexing-speed]]
+== Tune for indexing speed
```

Comment:
Is it better now that I moved them up or would you like other sections to be moved as well?